index.js

```diff
@@ -126,43 +126,53 @@ module.exports = {
       }, {
         name: 'splitAddress',
         label: 'Split Address',
-        type: 'checkboxes',
+        type: 'array',
         help: 'Optional: choose fields to split address to',
-        choices: [
-          {
-            value: 'street_number',
-            label: 'Street Number'
-          },
-          {
-            value: 'route',
-            label: 'Route'
-          },
-          {
-            value: 'locality',
-            label: 'City'
-          },
-          {
-            value: 'sublocality_level_1',
-            label: 'City (for NYC area)'
-          },
-          {
-            value: 'administrative_area_level_1',
-            label: 'State'
-          },
-          {
-            value: 'country',
-            label: 'Country'
-          },
+        titleField: 'addressPart',
+        schema: [
           {
-            value: 'postal_code',
-            label: 'Postal Code'
+            name: 'addressPart',
+            label: 'Address Part',
+            type: 'select',
+            choices: [{
+              value: 'street_number',
+              label: 'Street Number'
+            },
+            {
+              value: 'route',
+              label: 'Route'
+            },
+            {
+              value: 'locality',
+              label: 'City'
+            },
+            {
+              value: 'sublocality_level_1',
+              label: 'City (for NYC area)'
+            },
+            {
+              value: 'administrative_area_level_1',
+              label: 'State'
+            },
+            {
+              value: 'country',
+              label: 'Country'
+            },
+            {
+              value: 'postal_code',
+              label: 'Postal Code'
+            },
+            {
+              value: 'postal_town',
+              label: 'Postal Code (for UK and Sweden)'
+            }]
           },
           {
-            value: 'postal_town',
-            label: 'Postal Code (for UK and Sweden)'
+            name: 'label',
+            label: 'Label',
```

Comment:
Can't this be taken care of via i18n? Requiring the editor to type in all of these every time they add an address field feels excessive. Especially if they have a lot of them.

Also I am not sure that we really need to show the split fields to the end user all the time. Etienne did indicate interest, so that should be supported yes, but a boolean to determine whether the broken-out address fields are actually shown to the end user would be nice, for cases where the editor wants to make the form less intimidating to fill out.